modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -56,9 +56,7 @@ These files must be located at `src/main/resources`, inside your Mule project, o
 [[supported_files]]
 == Supported Files
 
-The Configuration Properties supports both YAML configuration files and Properties files. The recommended approach is to use a YAML configuration files, because it allows the addition of type validations and autocompletion.
-
-`String` is the only supported type.
+The Configuration Properties support both YAML configuration files and Properties files. `String` is the only supported type in both file formats. Use the format that is easier to read and edit for you.
```

Comment:
I don't think Properties should have an initial cap in "Properties files." I see it both ways in the Mulesoft docs.

Reply:
I suggest rewording the last sentence to "Use the format that you find easier to read and edit."